driver-core/src/main/java/com/datastax/driver/core/TableOptionsMetadata.java

```diff
@@ -23,9 +23,10 @@
 public class TableOptionsMetadata {
 
   private static final String COMMENT = "comment";
-  private static final String READ_REPAIR = "read_repair_chance";
-  private static final String DCLOCAL_READ_REPAIR = "dclocal_read_repair_chance";
-  private static final String LOCAL_READ_REPAIR = "local_read_repair_chance";
+  private static final String READ_REPAIR_CHANCE = "read_repair_chance";
```

Comment:
I renamed a few private variables:
`READ_REPAIR` to `READ_REPAIR_CHANCE`
`DCLOCAL_READ_REPAIR` to `DCLOCAL_READ_REPAIR_CHANCE`
`LOCAL_READ_REPAIR` to `LOCAL_READ_REPAIR_CHANCE`
so that I could introduce `READ_REPAIR`.

Luckily the public access methods were already `getXYZchance()`